### ISSUE TITLE
Mention class lacking implementation in `abstract-method` message

### ISSUE THREAD

**Issue by harupy** [PyCQA/pylint#7124]
### Bug description

```python
from abc import abstractmethod, ABCMeta


class WildAnimal(metaclass=ABCMeta):
    @abstractmethod
    def make_sound(self):
        pass


class Panther(WildAnimal):
    def make_sound(self):
        raise NotImplementedError("Panther")


class PantherChild(Panther):
    pass
```

pylint raise the following error against the code above?

```
a.py:15:0: W0223: Method 'make_sound' is abstract in class 'Panther' but is not overridden (abstract-method)
```

Is this a false positive?

### Command used

```shell
# Save the code above as a.py and then run:
docker run --rm -w /app -v $(pwd)/a.py:/app/a.py python:3.7 bash -c "pip install pylint && pylint --disable=C,R a.py"
```

### Pylint output

```shell
************* Module a
a.py:15:0: W0223: Method 'make_sound' is abstract in class 'Panther' but is not overridden (abstract-method)
```

### Expected behavior

pylint should not raise `abstract-method` because `make_sound` is overridden in `Panther`?

### Pylint version

```shell
3.7.13
```

**Comment by jacobtylerwalls** [comment]
It's not overridden in `PantherChild`, because it's reported on line 15. We could add the child class in the message output, though.

**Comment by harupy** [comment]
@jacobtylerwalls Thanks for the comment. Is `Panther.make_sound` considered an abstract method since it contains `raise NotImplementedError("Panther")`?

**Comment by jacobtylerwalls** [comment]
Exactly!

https://pylint.pycqa.org/en/latest/user_guide/messages/warning/abstract-method.html

**Comment by clavedeluna** [comment]
Happy to pick this up but just to verify, is the ask to change this output

 Method 'make_sound' is abstract in class 'Panther' but is not overridden (abstract-method)

to this?

 Method 'make_sound' is abstract in class 'Panther' but is not overridden in child class `PantherChild` (abstract-method)

?

**Comment by DanielNoord** [comment]
Yes! That makes the error a little bit more actionable!
